app/models/commit_monitor_branch.rb

```
@@ -31,4 +31,19 @@ def last_commit_uri
   def pr_number
     MiqToolsServices::MiniGit.pr_number(name) if pull_request?
   end
+
+  def write_github_comment(message, header, continuation_header = nil, replace = true)
+    return unless pull_request?
```

This should probably raise an exception if it's not a PR.